docs/pages/workflows/get-started.mdx

```diff
@@ -0,0 +1,143 @@
+---
+
+import { BookOpen02Icon } from '@expo/styleguide-icons/outline/BookOpen02Icon';
+
+import { BoxLink } from '~/ui/components/BoxLink';
+import { Callout } from '~/ui/components/Callout';
+import { Collapsible } from '~/ui/components/Collapsible';
+import { Terminal } from '~/ui/components/Snippet';
+
+<Callout type="info">
+  Workflows are currently in preview. We may introduce breaking changes in the coming weeks. Got
+  feedback? Send us an email at workflows@expo.dev.
+</Callout>
+
+Builds, submissions, updates and more are all a part of delivering your app to users. Workflows consist of a sequence of jobs, which help you and your team get things done. You can do things like building your project, run end-to-end tests, submit that build to the app stores, then run custom scripts after the submission is complete. Since each job can have prerequisites and conditionals, you can automate you and your team's release process.
+
+<Collapsible summary="How do workflows compare to other CI services?">
+
+Workflows on EAS are designed to help you and your team release your app. It comes preconfigured with pre-packaged job types that help you build, submit, update, run Maestro tests, and more. It all runs on EAS, so you'll only have to manage one set of YAML files, and all of the artifacts from your job runs will appear on expo.dev.
+
+Other CI services, like CircleCI and GitHub Actions, are more generalized and have the ability to do more than workflows. However, those services also require you to understand more about the implementation of each job. While that is necessary in some cases, workflows help you get common tasks done quickly by pre-packaging the most essential types of jobs for app developers. In addition, workflows are designed to provide you with the fastest possible cloud machine for the task at hand, and we're constantly updating those for you.
+
+Workflows are great for operations related to your Expo apps, while other CICD services will provide a better experience for other types of workflows.
+
+</Collapsible>
+
+## Set up your project
+
+If you haven't already, you'll need to create a project and sync it with EAS:
+
+<Collapsible summary="Create a project and sync it with EAS">
+  You can create a new project with the following command:
+
+{' '}
+
+<Terminal cmd={['$ npx create-expo-app@latest']} />
+
+Once you've created the project, login with your account:
+
+{' '}
+
+<Terminal cmd={['$ npx eas-cli@latest login']} />
+
+Finally, link the project you have locally with EAS:
+
+  <Terminal cmd={['$ npx eas-cli@latest init']} />
+</Collapsible>
+
+Then, create a directory named **.eas/workflows** at the root of your project with a **.yaml** file inside of it. For example: **.eas/workflows/hello-world.yaml**.
+
+## Write a workflow
+
+Each workflow consists of three top-level elements:
+
+- `name`: defines the name of the workflow. For example: "Hello World!"
```

Comment:
```suggestion
- `name`: defines the name of the workflow. For example: "Hello World"
```

Just to align with the example below, let's remove the exclamation mark from here.